src/combine.cpp

```diff
@@ -5740,7 +5739,17 @@ static void handle_proto_wrap(chunk_t *pc)
    set_chunk_parent(clp, pc->type);
 
    set_chunk_parent(tmp, CT_PROTO_WRAP);
-   fix_fcn_def_params(tmp);
+
+   if (tmp->type == CT_PAREN_OPEN)
+   {
+	  fix_fcn_def_params(tmp);
```

Comment:
Here I've made an indentation mistake (tab instead of space). It will eventually get fixed with other source formatting changes.